packages/ui/cypress/e2e/login/index.spec.js

```diff
@@ -0,0 +1,103 @@
+describe('Login', () => {
+  context('with login form', () => {
+    beforeEach(() => {
+      cy.visit('/login')
+    })
+
+    it('can login by pressing enter', () => {
+      cy.get('[name=username]')
+        .type('test')
+        .get('[name=password]')
+        .type('password{enter}')
+        .url()
+        .should('include', '/app')
+    })
+
+    it('can login by clicking login button', () => {
+      cy.get('[name=username]')
+        .type('test')
+        .get('[name=password]')
+        .type('password')
+        .get('[name=btn-login]')
```

Comment:
sorry I didn't think of this before, but let's exchange this name property on the button with a data-testid="login-button". As is commented in the login form that name was added long time ago for another testing thing. so from now on, we use data-testid when we need to add an identifier.